environments/production/app-processes.yml

```diff
@@ -35,6 +35,8 @@ celery_processes:
       concurrency: 134
     export_download_queue:
       concurrency: 5
+    linked_domain_queue:
+      concurrency: 20
```

Comment:
This is per machine on 6 machines, so this is 120 processes. I suspect you want something like 3 or 4

Reply:
Indeed, 3 sounds good. 4e29f1256